### ISSUE TITLE
Feature Enhancement: Change Symbols to Library Symbol Base Symbol ID (keep size)

### ISSUE THREAD

**Issue by ins** [Ashung/Automate-Sketch#74]
I want to use the *Change Symbols to Library Symbol Base Symbol ID*. **It works but resets all instances to the original size, which is a problem for me**.

**The feature I need is already coded** in the *Replace Symbol With Library Symbol* dialog. There is an option to *After replace, set all instances to original size*. If I use this, with the checkbox unchecked it's great, but doing it 1-by-1 manually is slow.

The simplest fix for me would be adding another command, that doesn't reset symbol size, maybe called *Change Symbols to Library Symbol Base Symbol ID (keep size)*.

I'm using latest Plugin (50.2.20180529) and Sketch 50.2 (55047)

**Comment by richardsison** [comment]
Hi there, this is marked as "Done" but isn't working for me.

When I select "Change Symbols to Library Symbol Base Symbol ID", all of the Local Symbols which get replaced with a Library Symbol resize to the original, source size (i.e. in the Library); I'd like ALL resized symbols in my local, working file to remain at the resized size.

When I used the "Replace Symbol With Library Symbol" option, like the original poster mentioned, the checkbox allows us to either keep at the current, resized size or reset the sizing.

Here are some examples:

1. [Replace Symbol With Library Symbol](https://d.pr/i/rAJQLu). Notice how the symbol relinks to the library symbol, but retains the size.

2. [Change Symbols to Library Symbol Base Symbol ID](https://d.pr/i/jniR92). Notice how once I've relinked the symbol, it changes to the original symbol size.
